config/env/experimental.env

```
@@ -13,4 +13,11 @@ DB_SSL_MODE=verify-full
 DB_SSL_ROOT_CERT=/bin/rds-combined-ca-bundle.pem
 DEVLOCAL_CA=/config/tls/devlocal-ca.pem
 DEVLOCAL_AUTH=false
-FEATURE_FLAG_ACCESS_CODE=false
\ No newline at end of file
+FEATURE_FLAG_ACCESS_CODE=false
+SERVE_ADMIN=true
+SERVE_SDDC=true
+SERVE_ORDERS=true
+SERVE_DPS=true
+SERVE_API_INTERNAL=true
+SERVE_API_EXTERNAL=true
+MUTUAL_TLS_ENABLED=true
```

Go ahead and drop `MUTUAL_TLS_ENABLED` from all of these `*.env` files.

---

I did notice with this that removing it causes acceptance tests to fail, with orders still in these files. Should I keep mutual_tls_enabled here or remove orders or something else in your view?

---

That's probably because of a templating issue. Instead leave it in but set it like this:

```suggestion
MUTUAL_TLS_ENABLED=false
```